src/configurations/destinations/intercom_v2/ui-config.json

```diff
@@ -0,0 +1,230 @@
+{
+  "uiConfig": {
+    "baseTemplate": [
+      {
+        "title": "Initial setup",
+        "note": "Review how this destination is set up",
+        "sections": [
+          {
+            "groups": [
+              {
+                "title": "Connection Settings",
+                "note": "Update your connection settings here",
+                "icon": "settings",
+                "fields": [
+                  {
+                    "type": "singleSelect",
+                    "label": "Intercom REST API Version",
+                    "configKey": "apiVersion",
+                    "options": [
+                      {
+                        "label": "latest",
+                        "value": "v2"
+                      }
+                    ],
+                    "default": "v2",
+                    "note": "Select your Intercom REST API Version to send data to intercom.",
+                    "preRequisites": {}
+                  },
+                  {
+                    "type": "singleSelect",
+                    "label": "API Server",
+                    "configKey": "apiServer",
+                    "options": [
+                      {
+                        "label": "Standard",
+                        "value": "standard"
```

Comment:
Rather than calling it standard lets share the details in the following format (US, Europe, Australia)
https://developers.intercom.com/docs/build-an-integration/learn-more/rest-apis#regional-hosting